README.md

```diff
@@ -29,8 +29,34 @@ To download:
 git clone https://github.com/IQTLabs/deps2repos
 ```
 
+Download Bioconda-utils and Bioconda-recipes (Only if attempting to parse Bioconda Recipes)
```

Comment:
@spearw, these are helpful instructions. Thank you! I do think they might overwhelm the user if they are in the main README though. Would you please create a separate, bioconda-specific README and link to that README. That way this main README is less cognitively demanding but has a link to the full explanation. Again, nice instructions!

Reply:
Good suggestion. Done.